src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -507,29 +578,27 @@ function discoverController(
     };
   };
 
-  $scope.uiState = $state.makeStateful('uiState');
+  function getDefaultQuery() {
```

Comment:
Could you please add this function as a utility to `src/plugins/data/public/query/lib` and export it on the export block `from './query';`?

Reply:
of course! been there, done that!